src/transport/components/split-get.js

```diff
@@ -0,0 +1,111 @@
+'use strict';
+
+var qutil = require('../../util/query-util');
+var MAX_URL_LENGTH = 2048;
+
+module.exports = function (httpOptions) {
+    return {
+        runAPI: function (params, options) {
```

Comment:
I'm not sure about having a generic name like `split-get` have specifics about `runApi`; I'd rather:

- Move this function to `run-utils` and call it splitGet
- Change the invocation from `http.add('splitGet', 'runAPI')` to `http.add(runtils.splitGet)` - this way the httpTransport itself can be agnostic to splitGets but still maintain the 'component structure' so calling services can extend it